### PR TITLE
Flatmap-viewer API Updates 

### PR DESCRIPTION
**Pull request by akhuoa**
This PR includes the updates for `flatmap-viewer` version. 

Some of the new API methods are `queryKnowledge()`, `queryLabels()`, and `queryPublications()`, and those are used instead of querying directly to the server.

The display nerves (centreline) have been removed.

The only error left is `schema-version`.

### CODE REVIEW

File: src/components/FlatmapVuer.vue [hunk 1]
```diff
@@ -1178,10 +1178,14 @@ export default {
     processTaxon: function (flatmapAPI, taxonIdentifiers) {
       this.taxonConnectivity.length = 0
       taxonIdentifiers.forEach((taxon) => {
-        findTaxonomyLabel(flatmapAPI, taxon).then((value) => {
-          const item = { taxon, label: value }
-          this.taxonConnectivity.push(item)
-        })
+        this.mapImp.queryLabels(taxon).then((entityLabels) => {
```
Comment (alan-wu) [inline src/components/FlatmapVuer.vue:1181]: This can api can take a lost of taxons now, please try that.

File: src/services/flatmapQueries.js [hunk 2]
```diff
@@ -68,12 +68,14 @@ let FlatmapQueries = function () {
...
       taxonomyLabel = []
       for (let i = 0; eventData.provenanceTaxonomy.length > i; i++) {
-        taxonomyLabel.push(
-          await findTaxonomyLabel(
-            this.flatmapAPI,
-            eventData.provenanceTaxonomy[i]
-          )
-        )
+        const taxon = eventData.provenanceTaxonomy[i];
+        mapImp.queryLabels(taxon).then((entityLabels) => {
```
Comment (alan-wu) [inline src/services/flatmapQueries.js:72]: Same as above.

File: src/components/FlatmapVuer.vue (hunk 1 above)
Comment (akhuoa) [reply src/components/FlatmapVuer.vue]: @alan-wu this is updated now.

File: src/services/flatmapQueries.js (hunk 2 above)
Comment (akhuoa) [reply src/services/flatmapQueries.js]: @alan-wu this is updated now.

**Review by alan-wu — APPROVED** [review]
Looks good to me.

**Pull request merged**